tests/unit_tests/TerminusTestCase.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Pantheon\Terminus\UnitTests;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class TerminusTestCase
+ * @package Pantheon\Terminus\UnitTests
+ */
+abstract class TerminusTestCase extends TestCase
```

Comment:
Empty base class is not needed. This could serve as a place to put future utility methods, but those could also be provided via traits, and inserted where needed / used. That said, though, doing it this way is fine.

Reply:
I did this because I don't want to edit 81 files the next time PHPUnit changes the name of their TestCase class.